### PR TITLE
fix: location icon doesn't change according to theme

### PR DESCRIPTION
**Pull request by Ryukemeister**
## What does this PR do?

The icon which specifies the location of the attendee in the preview component doesn't change as we change the theme

Screenshot 2023-09-22 at 6 18 42 PM

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

File: packages/features/bookings/components/event-meta/AvailableEventLocations.tsx [hunk 1]
```diff
@@ -22,7 +22,8 @@ function RenderIcon({
     <img
       src={eventLocationType.iconUrl}
       className={classNames(
-        eventLocationType?.iconUrl?.includes("-dark") && "dark:invert",
+        (eventLocationType?.iconUrl?.includes("-dark") || eventLocationType?.iconUrl === "/map-pin.svg") &&
+          "dark:invert",
```
Comment (Udit-takkar) [inline packages/features/bookings/components/event-meta/AvailableEventLocations.tsx:26]: @Ryukemeister pushed this change as there can be many icons for location depending on the location

**Review by Udit-takkar — DISMISSED** [review]
LGTM

File: packages/features/bookings/components/event-meta/AvailableEventLocations.tsx (hunk 1 above)
Comment (Ryukemeister) [reply packages/features/bookings/components/event-meta/AvailableEventLocations.tsx:26]: @Udit-takkar thanks, might've missed that!

File: README.md [hunk 2]
```diff
@@ -145,8 +145,8 @@ Here is what you need to be able to run Cal.com.
 
 1. Set up your `.env` file
    - Duplicate `.env.example` to `.env`
-   - Use `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
-   - Use `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
+   - Use the following command `openssl rand -base64 32` to generate a key and add it under `NEXTAUTH_SECRET` in the `.env` file.
+   - Use the following command `openssl rand -base64 24` to generate a key and add it under `CALENDSO_ENCRYPTION_KEY` in the `.env` file.
```
Comment (hariombalhara) [inline README.md:149]: @Ryukemeister Could you revert these? I think the earlier text makes more sense.

File: packages/features/bookings/components/event-meta/AvailableEventLocations.tsx (hunk 1 above)
Comment (hariombalhara) [reply packages/features/bookings/components/event-meta/AvailableEventLocations.tsx:26]: How about renaming the file to `map-pin-dark.svg`? That way there is no specific file name check here.

File: README.md (hunk 2 above)
Comment (Ryukemeister) [reply README.md:149]: @hariombalhara yeah will do that

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Thanks @Ryukemeister 🙏 Some things need to be fixed before it can be merged.

**Review by Udit-takkar — DISMISSED** [review]
LGTM

File: packages/features/bookings/components/event-meta/AvailableEventLocations.tsx [hunk 3]
```diff
@@ -98,9 +98,9 @@ export function AvailableEventLocations({ locations }: { locations: LocationObje
   return filteredLocations.length > 1 ? (
     <div className="flex flex-row items-center text-sm font-medium">
       <img
-        src="/map-pin.svg"
+        src="/map-pin-dark.svg"
         className={classNames("me-[10px] h-4 w-4 opacity-70 dark:invert")}
-        alt="map-pin"
+        alt="map-pin-dark"
```
Comment (hariombalhara) [inline packages/features/bookings/components/event-meta/AvailableEventLocations.tsx:103]:
```suggestion
        alt="map-pin"
```

I don't think alt needs to have dark in it. It is read by accessibility tools and doesn't have any significance there

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**